src/commands/memes/noHello.ts

```diff
@@ -14,7 +14,9 @@ export const noHello: CommandDefinition = {
             .then((res) => res.reply(NO_HELLO_URL))
             .catch(async () => msg.channel.send(NO_HELLO_URL));
 
-        msg.delete();
+        if (msg.content.split(/\s+|\n|\r|\.|-|>|\/|\\/).length === 1) {
```

Comment:
```suggestion
        if (msg.content.split(/\s+|\n|\r|\.|-|>|\/|\\/).length <= 2) {
```
Needs to be 2, because splitting on `.` automatically results in a list of 2 elements. Verified by adding (no need to add logging in PR, of course):
```
Logger.debug(`"${msg.content}" - ${msg.content.split(/\s+|\n|\r|\.|-|>|\/|\\/).length} - ${JSON.stringify(msg.content.split(/\s+|\n|\r|\.|-|>|\/|\\/))}`);
``` 
Output: 
```
debug: ".nohello" - 2 - ["","nohello"]
```